index.html

```diff
@@ -38,23 +38,16 @@
       <div class="inner">
         <nav>
           <ul>
+            <!--for these to do the fun swipe thing they have to only include
+            internal links. I have added a section for external links below.
+            This also does not show on mobile so we should be aware of that
+            when adding things here. Add it somewhere they can see-->
```

Comment:
Oh... even the template's [demo](https://html5up.net/hyperspace) doesn't have a navbar for mobile.  Yikes.

Can we decouple this comment from the PR by removing the one-off comment:
> I have added a section for external links below

Also, I'm going to nitpick, but can we capitalize the "F" in "for"?

Reply:
uh I am not quite sure what you mean by:
>Can we decouple this comment from the PR by removing the one-off comment:
and 
> Also, I'm going to nitpick, but can we capitalize the "F" in "for"?

which for?

Reply:
In L41,

```
<!--for
```

```
<!-- For
```

Reply:
In L42, there is a comment that's more about a specific change made:

> I have added a section for external links below.

It does not belong in the source code itself in my opinion, because it's pertaining to a PR.  Another way to look at it is: if another dev were to open up this file and read this comment later on, the previous sentence is enough to understand the intent of this navbar.

Reply:
all good, added in last commit